src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -311,8 +312,8 @@ private JToken TransformScalar(ErrorBuilder errors, JsonSchema schema, FilePath
                     {
                         // the content here must be an UID, not href
                         var (xrefError, xrefSpec, href) = _xrefResolver.ResolveXrefSpec(
-                            content, file, file, _monikerProvider.GetFileLevelMonikers(ErrorBuilder.Null, file));
-                        errors.AddIfNotNull(xrefError);
+                            content, file, file, _monikerProvider.GetFileLevelMonikers(ErrorBuilder.Null, file), propName);
```

Comment:
Add an `Error.WithName` method to create a new error in the translation stage to avoid having to change signatures for `ResolveXrefSpec`.

Reply:
OK